Libplanet.Stun/Stun/Messages/StunMessage.cs

```diff
@@ -123,28 +121,23 @@ public enum MessageMethod : ushort
                     MessageMethod.Binding => new BindingSuccessResponse(),
                     MessageMethod.CreatePermission => new CreatePermissionSuccessResponse(),
                     MessageMethod.Refresh => new RefreshSuccessResponse(),
-                    _ => rv,
+                    _ => throw new TurnClientException("Parsed result is null."),
```

Comment:
Wouldn't this change cause a change in behavior in other projects using Libplanet if they were written with the expectation that no exception would occur?

Reply:
Looking at the code below, if 'rv' is null, it raises an exception, so you can consider it the same code.